test/gpu/native/studies/sort/radixSortGpu.graph

```diff
@@ -1,5 +1,5 @@
-perfkeys: Time:, Time:, Time:
-files: serialScan.dat, hillisSteeleScan.dat, blellochScan.dat
-graphkeys: Serial, Hillis-Steele, Blelloch
+perfkeys: Time:, Time:, Time:, Time:, Time:
```

Comment:
I am not finding it my notes, so sheepish to ask that you remind me: how did you convince me to lump this new data onto the existing plot vs adding "Historical" to the old plot's title and adding a new one for this case?

It looks like there's almost no historical continuity between what we had and what you are adding here.

Reply:
Your point about continuity makes complete sense. I think we only briefly discussed this, and the point was that it shows that the previous algorithms were a predecessor to the current algorithms being used. I think the two options we discussed were either getting rid of the old graph and starting anew or keeping the old data and adding an annotation to the graph when we switch to the new algorithm.

This option of changing the name of the old graph so we don't get rid of it sounds good to me. It is going to be a stale graph at one point however since new data won't be added to it anymore.

Reply:
> This option of changing the name of the old graph so we don't get rid of it sounds good to me. It is going to be a stale graph at one point however since new data won't be added to it anymore.

Yeah, I think we can remove it altogether when it becomes more irrelevant.

Reply:
I've made this change and updated the OP with a new screenshot of what it'll look like.